build.gradle

```diff
@@ -1,83 +1,61 @@
 plugins {
-    id 'org.springframework.boot' version '2.5.1'
     id 'java'
+    id 'org.springframework.boot' version '2.5.5'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
 }
 
-apply plugin: 'io.spring.dependency-management'
-apply plugin: 'jacoco'
-
 group = 'uk.gov.digital.ho.hocs'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
-configurations {
-    compileOnly {
-        extendsFrom annotationProcessor
-    }
-}
-
 repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
 }
 
-task copyLombok(type: Copy) {
-	outputs.upToDateWhen { false }
-	into "${buildDir}/libs"
-	include "lombok*.jar"
-	from configurations.compileClasspath
-}
-
-build.dependsOn(copyLombok)
-
-repositories {
-	mavenCentral()
-    maven { url 'https://jitpack.io' }
+ext {
+    springCloudVersion = "Hoxton.SR12"
 }
 
 dependencies {
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.5'
-
-    implementation('org.springframework.boot:spring-boot-starter-web') {
+    implementation('org.springframework.boot:spring-boot-starter-web:2.5.5') {
         exclude(module: 'spring-boot-starter-tomcat')
     }
-    implementation('org.springframework.boot:spring-boot-starter-undertow')
-    implementation('net.logstash.logback:logstash-logback-encoder:5.3')
-    implementation('org.springframework.boot:spring-boot-starter-json')
-    implementation('org.springframework.boot:spring-boot-starter-actuator')
-    implementation('org.springframework.retry:spring-retry')
-    implementation('org.springframework:spring-aspects')
 
+    implementation 'org.springframework.boot:spring-boot-starter-undertow:2.5.5'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator:2.5.5'
+    implementation 'org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE'
+
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.6'
 
     // Override the default version of elasticSearch, we're still using V6
     def elasticSearchVersion = '6.8.3'
```

Comment:
Inlining this will help going forward and tell you if there is updates...

Reply:
done